clients/admin-ui/src/features/config-wizard/config-wizard.slice.ts

```diff
@@ -51,89 +43,13 @@ export const slice = createSlice({
       // Ensure the step number stays in the valid range.
       if (draftState.step < 1) {
         draftState.step = 1;
-      } else if (STEPS.length <= draftState.step) {
+      } else if (STEPS.length < draftState.step) {
```

Comment:
this was a bug: step numbers are _not_ zero indexed, so we don't need to do `<=`, and need to instead do `<`. this was never apparent before because we always had a step 6 "view your datamap" which we never hit